pages/running/index.vue

```diff
@@ -1,11 +1,20 @@
 <template>
   <section class="docs-container container">
-    <h1>Running a Game</h1>
-    <api-results-table v-if="sections" :data="sections" endpoint="running" />
+    <div class="filter-header-wrapper">
+      <h1 class="filter-header">Running a Game</h1>
+    </div>
+    <api-results-table
+      endpoint="running"
+      :api-endpoint="API_ENDPOINTS.sections"
+      :cols="['document__title', 'document__slug']"
+    />
   </section>
 </template>
 
 <script setup>
 import ApiResultsTable from '~/components/ApiResultsTable.vue';
-const { data: sections } = useSections('Rules');
+
+const filter = ref({
+  parent__in: 'Rules',
```

Comment:
This also seems to be returning a list of all sections instead of just the ones from the sidebar list